app/services/hyku_addons/entry_validation_service.rb

```diff
@@ -20,14 +20,19 @@ class EntryValidationService
       _version_ timestamp thumbnail_path_ss date_modified_dtsi system_create_dtsi system_modified_dtsi
       accessControl_ssim abstract_oai_tesim account_cname_tesim actionable_workflow_roles_ssim disable_draft_doi_tesim
       nesting_collection__deepest_nested_depth_isi nesting_collection__pathnames_ssim official_link_tesim doi_oai_tesim
-      draft_doi_tesim file_availability_tesim work_tenant_url_tesim edit_access_person_ssim
+      draft_doi_tesim file_availability_tesim work_tenant_url_tesim edit_access_person_ssim nesting_collection__ancestors_ssim
```

Comment:
Would you mind putting `nesting_collection__ancestors_ssim` on a new line? 
![image](https://user-images.githubusercontent.com/1199977/115895787-b6700400-a452-11eb-9c82-ff9c8165b9b8.png)